docs/usage.md

```diff
@@ -5,90 +5,225 @@
 
-Refer to the section `OT - Asset Management` to understand how this solution pack automation addresses your needs.
+- [OT - Add Sample Assets](#ot---add-sample-assets)
 
-# OT- Asset Management
+- [OT - Add Sample Alerts](#ot---add-sample-alerts)
 
-### Step to be followed after installing SP:
+- [OT - Stuxnet Attack Scenario](#ot---stuxnet-attack-scenario)
 
-1. Ingest MITRE **Group**, **Tactics**, **Techniques**, **Sub-techniques**, **Mitigations**, **Software**, and **Mitigation** data using `MITRE ATT&CK` connector on `ICS` configuration.
+- [OT - Asset Change Activity Scenario](#ot---asset-change-activity-scenario)
 
-> **NOTE**: On alert creation, IOCs are enriched, and the alerts are correlated with Techniques, Mitigation, and all other records for the MITRE ATT&CK Id available in the alert.
+Refer to the subsequent sections to understand how this solution pack's automation addresses your needs.
 
-## Simulation mode
+<table>
+    <tr>
+        <td><strong>IMPORTANT</strong></td>
+        <td>For seamless execution of included scenarios, set the global variable <code>Demo mode</code> to <code>true</code>.</td>
+    </tr>
+</table>
 
-The simulation mode has some sample data that helps you get a better understanding of how the solution pack functions. Following steps help you use the solution pack with some included sample data.
+## OT - Add Sample Assets
 
-1. **OT - Add Sample Assets**
+This scenario generates 87 IT/OT assets as per the Purdue model based on various criticality levels, types, and other asset categorizations as sample data.
 
-    - Browse to `Simulations` > `OT - Add Sample Assets` scenario and click **Simulate Scenario**.
+To run this scenario and generate the assets, perform the following steps:
 
-    - This scenario adds 86 different sample OT Assets (different Levels, Types etc.) for testing dashboards, use case playbooks, reports and other actions.<br>
+1. Under the FortiSOAR&trade; menu select to open **Simulations**.
 
-2. **OT - Add Sample Alerts**
+2. Select the scenario **OT - Add Sample Assets**.
 
-    - Browse to `Simulations` > `OT - Add Sample Alerts` scenario and click **Simulate Scenario**.
+3. Click **Simulate Scenario**.
 
-    - This scenario adds 12 well populated sample OT Alerts for testing dashboards, use case playbooks, reports and other actions.<br>
+4. To view the added sample assets, under the FortiSOAR&trade; menu select to open **Resources** > **Assets**.
 
-3. **OT - Stuxnet Attack Scenario**
+![](./res/added-sample-assets.png)
 
-    - Make sure the global variable **Demo mode** is set to `true`. 
+## OT - Add Sample Alerts
 
-    - Browse to `Simulations` > `OT - Stuxnet Attack Scenario` scenario and click **Simulate Scenario**.
+This scenario generates 12 well-populated alerts associated with assets, generated in the previous scenario, as sample data.

```

Comment:
......associated with assets, generated with scenario

Reply:
This scenario generates 12 well-populated alerts associated with assets, generated in the [OT - Add Sample Assets](#ot---add-sample-assets) scenario, as sample data.